### PR TITLE
Remove FHIR Flag checks

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
## Description
Remove checks for any FHIR flag, global or individual. 

Another PR will remove the Admin UI flag control and the flag entry from the feature flag table.

Minor updates:
- Updates footer URLs on the HARP support page.
- Logs the harp session revocation event at info, instead of severe, level.

## Checklist

- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [ ] Tests are included and test edge cases.
- [ ] Tests have been run locally and pass.

## Screenshots (if appropriate)

- [ ] None applicable

### CODE REVIEW

File: src/main/resources/MAT.properties [hunk 1]
```diff
@@ -17,8 +17,8 @@ mat.password.expiry.email.subject="Your MAT Account "
 mat.from.emailAddress=sb-mat-noreply-help@hcqis.org
 mat.support.emailAddress=sb-mat-help@semanticbits.com
 
-mat.privacypolicy.url=http://www.cms.gov/About-CMS/Agency-Information/Aboutwebsite/Privacy-Policy.html
-mat.termsofuse.url=https://www.emeasuretool.cms.gov/web/guest/terms-of-use
+mat.privacypolicy.url=https://www.cms.gov/About-CMS/Agency-Information/Aboutwebsite/Policiesforaccessibility
+mat.termsofuse.url=https://harp.cms.gov/login/terms-of-use
 mat.accessibilitypolicy.url=https://www.emeasuretool.cms.gov/web/guest/accessibility-statement
```
Comment (RohitKandimalla) [inline src/main/resources/MAT.properties:22]: AccessibilityPolicy isn't working? I know its not a part of change, but I wonder if we have to fix it as well

**Review by RohitKandimalla — DISMISSED** [review]

File: src/main/resources/MAT.properties (hunk 1 above)
Comment (jkotanchik-SB) [reply src/main/resources/MAT.properties]: Nice catch! I had updated all three, or so I thought. Turns out I pasted over the updated privacy policy with the updated accessibility policy link.

**Review by RohitKandimalla — APPROVED** [review]
